Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -435,6 +440,8 @@ final class PurchasesOrchestrator {
                         Logger.rcPurchaseSuccess(Strings.purchase.purchased_product(
                             productIdentifier: productIdentifier
                         ))
+
+                        self.postPaywallEventsIfNeeded()
```

Comment:
This is called after a successful SK1 purchase. I think every SK1 purchase made using the SDK goes through this path, but I'd appreciate if there's somewhere else I missed.